openeo/util.py

```diff
@@ -375,6 +375,21 @@ def deep_set(data: dict, *keys, value):
         raise ValueError("No keys given")
 
 
+def guess_format(outputfile: str = None):
```

Comment:
In this context, I would just use `filename` as argument name, it's not necessary to assume it is an _output_ file

also, the default value `None` does not make sense (and won't work  anyway)

Reply:
Should I add GTIFF as a default or no default at all?

Reply:
I meant that  argument `outputfile`  should have no default value